weed/shell/command_volume_fsck.go

```diff
@@ -61,7 +64,8 @@ func (c *commandVolumeFsck) Do(args []string, commandEnv *CommandEnv, writer io.
 	verbose := fsckCommand.Bool("v", false, "verbose mode")
 	findMissingChunksInFiler := fsckCommand.Bool("findMissingChunksInFiler", false, "see \"help volume.fsck\"")
 	findMissingChunksInFilerPath := fsckCommand.String("findMissingChunksInFilerPath", "/", "used together with findMissingChunksInFiler")
-	applyPurging := fsckCommand.Bool("reallyDeleteFromVolume", false, "<expert only> delete data not referenced by the filer")
+	applyPurging := fsckCommand.Bool("purge", false, "<expert only!> after detection, delete missing data from volumes / delete missing file entries from filer")
```

Comment:
I want to keep the "ugly" long name "reallyDeleteFromVolume" so that it is clear what it does and that it is a dangerous command.

Just "purge" or "purgeAbsent", I can not tell what they do.

Reply:
I thought this nomenclature would be more obvious ("purge" the "missing chunks in filer"), but it seems like it could be just me.

So absolutely, lets go with what you prefer.

Reply:
"-reallyDeleteFilerEntries" sounds good enough to scare non-experts. :)

Reply:
Cheers to to even scarier flags in the future! :)